rotkehlchen/db/schema.py

```diff
@@ -289,7 +289,8 @@
     xpub TEXT NOT NULL,
     derivation_path TEXT NOT NULL,
     label TEXT,
-    PRIMARY KEY (xpub, derivation_path)
+    blockchain TEXT DEFAULT('BTC'),
+    PRIMARY KEY (xpub, derivation_path, blockchain)
```

Comment:
You are adding a new column here. You need to create a database migration for this. Also I believe we don't use defaults for anything in the schemas?

Reply:
yes you don't need a default here since it's a new column. Remove it. And add it to the existing DB upgrade.